great-idea/services.html

```diff
@@ -1,107 +1,129 @@
 <!doctype html>
+  <div class="container">
+    <html lang="en">
+    <head>
+      <meta charset="utf-8">
 
-<html lang="en">
-<head>
-  <meta charset="utf-8">
+      <title>Great Idea!</title>
 
-  <title>Great Idea!</title>
+      <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
+      <link rel="stylesheet" href="css/index.css">
 
-  <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+      <!--[if lt IE 9]>
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+      <![endif]-->
+    </head>
 
-  <!--[if lt IE 9]>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
-  <![endif]-->
-</head>
+    <body>
 
-<body>
+      <nav>
+        <a href="services.html">Services</a> 
+        <a href="#">Product</a>
+        <a href="#">Vision</a>
+        <a href="#">Features</a>
+        <a href="#">About</a>
+        <a href="#">Contact</a>
+        <div>
+          <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+        </div>
+      
+      </nav>
 
+      <div>
+        <img class="services-header-img" src="img/services-header.jpg" alt="Our services header image">
+      </div>
 
-Services
-Product
-Vision
-Features
-About
-Contact
+    
+      <div>
+        <h1>Services</h1>
 
-<img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+        <p>Our services provide the best digital value in nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque eleifend. Mauris euismod facilisis iaculis.</p>
+      </div>
 
+    <div class="services-content">
```

Comment:
so heckin close. you set this html up perfectly. all you need here is a display:flex on the services-content class, along with flex-wrap:wrap